samples/servicecontrol/fix-messages/Core_9/MessageRepairingEndpoint/Program.cs

```diff
@@ -0,0 +1,25 @@
+using System;
+using System.Threading.Tasks;
+using NServiceBus;
+
+class Program
+{
+    static async Task Main()
+    {
+        Console.Title = "MessageRepairingEndpoint";
+        var endpointConfiguration = new EndpointConfiguration("FixMalformedMessages.MessageRepairingEndpoint");
+        endpointConfiguration.UseSerialization<SystemJsonSerializer>();
+        endpointConfiguration.UseTransport<LearningTransport>();
+        endpointConfiguration.EnableInstallers();
+        endpointConfiguration.SendFailedMessagesTo("error");
```

Comment:
None of this is needed for the learning transport but perhaps we want to set the error queue explicitly to make it more clear what is going on since this is about dealing with failed messages?